tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -29,9 +29,9 @@
 metadata_1_versions = [str(LATEST_0_10_0)]
 metadata_2_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)]
 # we can add the following versions to `backward_compatible_metadata_2_versions` after the corresponding
-# bug-fix release 0.10.1.2, 0.10.2.2, 0.11.0.3, 1.0.2, and 1.1.1 are available:
-# str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)
-backward_compatible_metadata_2_versions = []
+# bug-fix release 0.10.1.2 is available:
+# str(LATEST_0_10_1),
+backward_compatible_metadata_2_versions = [str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)]
```

Comment:
Similar to #5605 , could we just use `metadata_2_versions - LATEST_0_10_1` here?

Reply:
Can't say for sure as I'm not the original author of the test, but I would say no.
I'll update the comment like I did in #5605 

@mjsax any thoughts on this?

Reply:
Yeah, the value is actually the same, it is a nit suggestion to make it obvious that "backward_compatible_metadata_2_versions is just one version (LATEST_0_10_1) less than metadata_2_versions, and it will be replaced by the latter once we have 0.10.1.3".

I think just adding the comment should be fine as well, same for #5605.

Reply:
Both should be fine -- not sure if the change improves anything -- but would not make it worth either.